### PR TITLE
generate report for a single course

### PR DESCRIPTION
**Pull request by maneeshatejani1**
generate course report for selected course

### CODE REVIEW

File: lms/templates/header/navbar-authenticated.html [hunk 1]
```diff
@@ -52,6 +52,14 @@
           </a>
       </div>
     % endif
+    % if user.is_staff:
+      <div class="mobile-nav-item hidden-mobile nav-item nav-tab">
+        <a class="tab-nav-link discover-new-link" href="/admin_dashboard"
```
Comment (mumarkhan999) [inline lms/templates/header/navbar-authenticated.html:57]: Kindly, use `reverse` to make url

File: openedx/features/wikimedia_features/admin_dashboard/settings/common.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+"""Common settings for Admin Dashboard"""
```
Comment (mumarkhan999) [inline openedx/features/wikimedia_features/admin_dashboard/settings/common.py:2]: To keep the things `consistent`, kindly add docstrings in other newly added files as well.
```
"""
My docstring here.
"""
And follow this pattern

File: openedx/features/wikimedia_features/admin_dashboard/static/admin_dashboard/js/course_reports.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+(function() {
+    'use strict';
+    var getCookie, ReportDownloads;
```
Comment (mumarkhan999) [inline openedx/features/wikimedia_features/admin_dashboard/static/admin_dashboard/js/course_reports.js:3]: Kindly try to use `let` in place of `var` as much as possible to keep the scope of variables limited.

File: openedx/features/wikimedia_features/admin_dashboard/templates/course_report/course-reports.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  <select name="course_categories" class="filter-selectbox">
+    %for course in courses:
+      <p>course</p>
+      <option value=${course}>${course.display_name_with_default}</option>
+    %endfor
+  </select>
+
+  <br><br><br>
+
+  <p><input type="button" name="list-profiles-csv" value="${_("Download profile information as a CSV")}" data-endpoint="${ section_data['key']['get_students_features_url'] }" data-csv="true"></p>
```
Comment (mumarkhan999) [inline openedx/features/wikimedia_features/admin_dashboard/templates/course_report/course-reports.html:32]: Kindly, indent this to increase the readability 
```
<p>
    <input ....>
        ....
    </input>
</p>
```

File: openedx/features/wikimedia_features/admin_dashboard/course_reports.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+from lms.djangoapps.courseware.access import has_access
+from lms.djangoapps.courseware.courses import get_course_by_id, get_courses
+from lms.djangoapps.instructor.views.instructor_dashboard import _section_data_download
+
+
+@login_required
+@ensure_csrf_cookie
+@cache_if_anonymous()
+def course_reports(request):
+    courses_list = []
```
Comment (mumarkhan999) [inline openedx/features/wikimedia_features/admin_dashboard/course_reports.py:17]: Not commenting anything on this function as I assume you are using it just for testing purposes.

File: openedx/features/wikimedia_features/admin_dashboard/static/admin_dashboard/js/course_reports.js (hunk 3 above)
Comment (maneeshatejani1) [reply openedx/features/wikimedia_features/admin_dashboard/static/admin_dashboard/js/course_reports.js:3]: sure, taken note

**Review by mumarkhan999 — APPROVED** [review]

**Pull request merged**